lib/clearance/authorization.rb

```diff
@@ -19,7 +19,7 @@ module Authorization
     #     end
     def require_login
       unless signed_in?
-        deny_access
+        deny_access(I18n.t('flashes.failure_when_access_denied'))
```

Comment:
In this case, we're specifically denying access because there is no signed in user. I don't think "failure_when_access_denied" is an appropriate key here because you can actually call `deny_access` anywhere for any reason. It's a public method.

What about something like, "failure_when_not_signed_in" or something?